repo_resource/common.py

```diff
@@ -376,6 +376,24 @@ def __manifest_out(self, filename):
         finally:
             self.__restore_oldpwd()
 
+    def resolve_manifest_include(self, manifest):
+        for idx, element in enumerate(list(manifest.iter())):
+            if element.tag == 'include':
+                sub_manifest_name = element.get('name')
+                sub_xml = ET.parse('.repo/manifests/'+sub_manifest_name)
```

Comment:
What happens when the include points towards a file that does not exist?

How does the error look like?

If the error message is unclear, please add some error handling in here to make sure that the caller has a proper error message

Reply:
By default that will return this kind of errors:
```
fatal: manifest 'aosp_include_invalid_project.xml' not available
fatal: include [/tmp/repo-resource-cache/.repo/manifests/]aosp_invalid.xml doesn't exist or isn't a file
```

Is it necessary for you ?